id_pkg/src/id_parse.py

```diff
@@ -9,20 +9,37 @@ class IdParse(LogParse):
     def __init__(self, syslog_file):
         self.syslog_to_dataframe(syslog_file)
 
-    def has_ip_spoofing(self):
```

Comment:
It looks like maybe you were doing some testing and commented out some code?  Can you please restore this to the original state?